servers/biometric_signal_sensor_interface/src/anomaly_detector/vt_helper.py

```diff
@@ -180,35 +180,37 @@ def delete_data(self):
         5 min buffer where nothing is deleted
         create a three more threads maybe?
         """
-        limit = 1000
-        ecg_limit = 25000
-
-        if len(self.ecg_dict) > ecg_limit:
-            ecg_lim = 2500
-            for key in sorted(self.ecg_dict):
-                if ecg_lim != 0:
-                    del self.ecg_dict[key]
-                    ecg_lim = ecg_lim - 1
-                else:
-                    break
-
-        if len(self.rr_dict) > limit:
-            rr_lim = 200
-            for key in sorted(self.ecg_dict):
-                if rr_lim != 0:
-                    del self.ecg_dict[key]
-                    rr_lim = rr_lim - 1
-                else:
-                    break
-
-        if len(self.hr_dict) > limit:
-            hr_lim = 200
-            for key in sorted(self.ecg_dict):
-                if hr_lim != 0:
-                    del self.ecg_dict[key]
-                    hr_lim = hr_lim - 1
-                else:
-                    break
+        print("DELETE called")
+        return
+        # limit = 1000
```

Comment:
try to avoid leaving commented out code.
if you need to temporarily disable something, just delete it.
Then you can copy the deleted code from a previous commit.